packages/fe-fpm-writer/test/unit/prompts/prompts.test.ts

```diff
@@ -246,6 +247,20 @@ describe('Prompts', () => {
             expect(result.viewOrFragmentPath.filePathProps?.fileName).toBe('Main.view.xml');
         });
 
+        test.each(types)('Type "%s", get code snippet, min ui5Version = 1.95.0', async (type: PromptsType) => {
```

Comment:
could you please add test to version like `1.96.34` or there is already such test?

Reply:
Added cases 1.96.0 - context path applied;
1.96.25 - no context path for filter or table building blocks